azext_iot/common/shared.py

```diff
@@ -214,3 +214,12 @@ class AuthenticationType(Enum):
 
     keyBased = "key"
     identityBased = "identity"
+
+
+class RenewKeyType(Enum):
+    """
+    Type of the RegenerateKey for the device.
+    """
+    Primary = "primary"
```

Comment:
Let's keep the single word attribute names lowercase to fit with the convention of the rest of the enum's in this module.

Reply:
Fixed